_pytest/main.py

```diff
@@ -96,7 +96,7 @@ def wrap_session(config, doit):
                     excinfo.value, pytest.exit.Exception):
                 excinfo = _pytest._code.ExceptionInfo()
```

Comment:
Yes, my bad, that's a left from over from a different approach. Please delete that line.